OpenRA.Game/Platform.cs

```diff
@@ -85,11 +85,18 @@ public static string OperatingSystem
 			{
 				if (CurrentPlatform == PlatformType.Linux)
 				{
+					var desktopType = Environment.GetEnvironmentVariable("XDG_CURRENT_DESKTOP");
 					var sessionType = Environment.GetEnvironmentVariable("XDG_SESSION_TYPE");
-					if (!string.IsNullOrEmpty(sessionType))
-						sessionType = $" ({sessionType})";
+
+					string suffix;
```

Comment:
```suggestion
					var suffix = string.Empty;
```
and then drop the `else` case.

Reply:
I think the current setup makes more sense. (Why assign values which will get overwritten anyway?)